stack/__main__.py

```diff
@@ -29,6 +29,10 @@
 )
 ACCESS_LEVELS = ('test', 'standard', 'full')
 
+SampleMetadataAccessorMembership = namedtuple(
+    'SampleMetadataAccessorMembership', ['name', 'service_account', 'permissions']
```

Comment:
The second entry is really a group, not just a single account, right?

Reply:
I've renamed this `member_key` to more closely align with Pulumi.